documentation/govuk-paas.md

```diff
@@ -134,22 +121,40 @@ cd /teacher-vacancy
 cf7 run-task <app_name> -c "rails task:name"
 ```
 
-## Deploy to dev
-Make sure you are logged in `teaching-vacancies-dev` space.
+## Deploy to dev or staging via commandline
+This builds and deploys a Docker image from local code.
 
 ```bash
-cf7 push -f paas/web/manifest-dev.yml
-cf7 push -f paas/worker/manifest-dev.yml
+make <PaaS-space> deploy-local-image
 ```
+performs these steps:
+
+- Builds and tags a Docker image from local code
+- Pushes the image to Docker Hub
+- Deploys the Docker image to PaaS
 
-## Deploy to staging
-Make sure you are logged in `teaching-vacancies-staging` space.
+You need:
+- Write access to Docker Hub `dfedigital/teaching-vacancies` repository. Ask in #digital-tools-support should you require it.
+- `SpaceDeveloper` role in the paas space you want to deploy to
+- Log in to Docker Hub and GOV.UK PaaS in your terminal
 
 ```bash
-cf7 push -f paas/web/manifest-staging.yml
-cf7 push -f paas/worker/manifest-staging.yml
+make dev deploy-local-image # Deploy to dev
+make staging deploy-local-image # Deploy to staging
 ```
 
+## Deploy to dev or staging via a commit to GitHub
```

Comment:
Very nice work @nickromneydfe and @saliceti if for some reasons you have a slow connection this will save a lot of time and bandwidth